### PR TITLE
2.10 - Category and group restrictions.

### PR DESCRIPTION
**Pull request by cfwdall**
New category & group restrictions in the admin page.

### CODE REVIEW

File: phpmyfaq/lang/language_en.php [hunk 1]
```diff
@@ -565,12 +565,13 @@
...
 $LANG_CONF["main.enableUserTracking"] = array(0 => "checkbox", 1 => "Enable user tracking");
 $LANG_CONF["main.enableAdminLog"] = array(0 => "checkbox", 1 => "use Adminlog?");
+$LANG_CONF["main.enable_category_restrictions"] = array(0 => "checkbox", 1 => "Enable category restrictions");
 $LANG_CONF["security.ipCheck"] = array(0 => "checkbox", 1 => "Check the IP in administration");
 $LANG_CONF["records.numberOfRecordsPerPage"] = array(0 => "input", 1 => "Number of displayed topics per page");
 $LANG_CONF["records.numberOfShownNewsEntries"] = array(0 => "input", 1 => "Number of news articles");
 $LANG_CONF['security.bannedIPs'] = array(0 => "area", 1 => "Ban these IPs");
 $LANG_CONF["main.enableRewriteRules"] = array(0 => "checkbox", 1 => "Enable URL rewrite support? (default: disabled)");
-$LANG_CONF["ldap.ldapSupport"] = array(0 => "checkbox", 1 => "Enable LDAP support? (default: disabled)");
+$LANG_CONF["security.ldapSupport"] = array(0 => "checkbox", 1 => "Enable LDAP support? (default: disabled)");
```
Comment (thorsten) [inline phpmyfaq/lang/language_en.php]: This change is wrong for 2.10

File: phpmyfaq/lang/language_en.php [hunk 2]
```diff
@@ -1249,22 +1250,4 @@
 
 // added v2.9.0-RC - 2016-02-19 by Thorsten
 $PMF_LANG['ad_admin_notes'] = 'Private Notes';
-$PMF_LANG['ad_admin_notes_hint'] = '%s (only visible for editors)';
```
Comment (thorsten) [inline phpmyfaq/lang/language_en.php]: Please don't remove these new strings for 2.10

File: phpmyfaq/lang/language_en.php [hunk 3]
```diff
@@ -3,7 +3,7 @@
 /**
  * The English language file - try to be the best of British and American English
  *
- * PHP Version 5.6
+ * PHP Version 5.5
```
Comment (thorsten) [inline phpmyfaq/lang/language_en.php]: It's PHP 5.6 at least.

File: phpmyfaq/admin/category.add.php [hunk 4]
```diff
@@ -93,31 +112,8 @@
                             </div>
                         </div>
                     </div>
-
-                    <div class="form-group">
```
Comment (thorsten) [inline phpmyfaq/admin/category.add.php]: Please don't remove this category image upload feature

File: phpmyfaq/admin/category.edit.php [hunk 5]
```diff
@@ -105,28 +122,6 @@ class="form-control">
                 </div>
             </div>
 
-          <div class="form-group">
```
Comment (thorsten) [inline phpmyfaq/admin/category.edit.php]: Please don't remove this category image upload feature

File: phpmyfaq/admin/category.main.php [hunk 6]
```diff
@@ -55,13 +54,6 @@
     $csrfCheck = true;
 }
 
-//
```
Comment (thorsten) [inline phpmyfaq/admin/category.main.php]: Please don't remove this category image upload feature

**Comment by cfwdall** [comment]
created a new pull request under 2.10 branch

**Pull request closed without merging**